demo/src/daisyui_demo/button.rs

```diff
@@ -1,12 +1,51 @@
 use daisyui::prelude::*;
 use yew::prelude::*;
 
+#[derive(Properties, PartialEq, Clone)]
+pub struct DisplayProp {
+    title: &'static str,
+    // code : &'static str,
```

Comment:
Remove this commented code for now. We will use what yew print is doing to show case code.